doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -51,4 +51,4 @@ And you can upload one manually:
 
 ## Package Release
 
-See [Release Checklist](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/8/Release-Checklist?anchor=prepare-release-script).
\ No newline at end of file
+See [Release Checklist](https://aka.ms/azsdk/release-checklist).
```

Comment:
@maririos -- would you be able to confirm this is the correct link for this documentation?

Reply:
@annelo-msft the current link points to a wiki which has this link. This is how I came up with this change. So I don't think it requires a confirmation from @maririos  :)

Reply:
The best link will actually be https://aka.ms/azsdk/releases/partnerinfo so that people can understand the process and the guidelines and then look at the check list.

Looking at the tittle though, why would a quickstart talk about releasing? 
It might be better if the document will stop at code review. Yay you created your first PR. There are multiple tasks that need to be completed in order to release an SDK. To get a comprehensive list of tasks that adapt to your specific product, create a [Release Plan](https://eng.ms/docs/products/azure-developer-experience/plan/release-plan) and follow the tasks in your plan.